docs/source/user_guide/timeseries.ipynb

```diff
@@ -309,6 +314,34 @@
     "pl.predict_in_sample(X_test, y_test, objective=None, X_train=X_train, y_train=y_train)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Validating the holdout data\n",
+    "\n",
+    "Before we predict on our holdout data, validating that it meets the requirements we summarized in the second point above in **Predicting on sunseen data** is important. We can call on `validate_holdout_datasets` in order to verify the two requirements:\n",
```

Comment:
nit: "Before we predict on our holdout data, validating that it meets the requirements we summarized in the second point above" ----> ""Before we predict on our holdout data, it is import to validate that it meets the requirements we summarized in the second above in ..."